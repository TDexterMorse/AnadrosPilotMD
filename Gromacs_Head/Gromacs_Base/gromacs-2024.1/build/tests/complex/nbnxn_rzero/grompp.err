                      :-) GROMACS - gmx grompp, 2024.1 (-:

Executable:   /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/bin/gmx
Data prefix:  /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1 (source tree)
Working dir:  /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/complex/nbnxn_rzero
Command line:
  gmx grompp -f /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/complex/nbnxn_rzero/grompp.mdp -c /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/complex/nbnxn_rzero/conf -r /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/complex/nbnxn_rzero/conf -p /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/complex/nbnxn_rzero/topol -maxwarn 10

Ignoring obsolete mdp entry 'optimize-fft'
Ignoring obsolete mdp entry 'rlistlong'
Ignoring obsolete mdp entry 'nstcalclr'
Ignoring obsolete mdp entry 'gb-algorithm'
Ignoring obsolete mdp entry 'nstgbradii'
Ignoring obsolete mdp entry 'rgbradii'
Ignoring obsolete mdp entry 'gb-epsilon-solvent'
Ignoring obsolete mdp entry 'gb-saltconc'
Ignoring obsolete mdp entry 'gb-obc-alpha'
Ignoring obsolete mdp entry 'gb-obc-beta'
Ignoring obsolete mdp entry 'gb-obc-gamma'
Ignoring obsolete mdp entry 'gb-dielectric-offset'
Ignoring obsolete mdp entry 'sa-algorithm'
Ignoring obsolete mdp entry 'sa-surface-tension'
Ignoring obsolete mdp entry 'ns-type'
Replacing old mdp entry 'verlet-buffer-drift' by 'verlet-buffer-tolerance'
Replacing old mdp entry 'nstxtcout' by 'nstxout-compressed'
Replacing old mdp entry 'xtc-precision' by 'compressed-x-precision'
Number of degrees of freedom in T-Coupling group rest is 9.00
The integrator does not provide a ensemble temperature, there is no system ensemble temperature

Thanx for Using GROMACS - Have a Nice Day

