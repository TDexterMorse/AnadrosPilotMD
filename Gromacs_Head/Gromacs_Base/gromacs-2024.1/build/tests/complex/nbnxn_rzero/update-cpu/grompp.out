Generated 2 of the 3 non-bonded parameter combinations

Excluding 1 bonded neighbours molecule type 'DIP'
Analysing residue names:
There are:     2      Other residues
Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...

This run will generate roughly 0 Mb of data
